include/fmt/chrono.h

```diff
@@ -2075,12 +2140,15 @@ template <typename Char> struct formatter<std::tm, Char> {
     return end;
   }
 
-  template <typename FormatContext>
-  auto format(const std::tm& tm, FormatContext& ctx) const
+  template <typename TimeType, typename FormatContext,
+            FMT_ENABLE_IF(std::is_same<TimeType, std::tm>::value ||
+                          std::is_same<TimeType, detail::tm_ss>::value)>
+  auto format(const TimeType& tm, FormatContext& ctx) const
```

Comment:
Let's keep the first argument as `std::tm` and pass subseconds as an additional argument.

Reply:
done